base/commandLine.cpp

```diff
@@ -1917,10 +1925,27 @@ bool processSettings(Common::String &command, Common::StringMap &settings, Commo
 		nullptr
 	};
 
+	// Skip some settings that should only be used for the command-line commands
+	static const char * const skipSettings[] = {
```

Comment:
I wonder, how we can make sure that in the future if such commands are added, they get into this list as well...